cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -387,6 +388,23 @@ def test_export_course_with_metadata_only_word_cloud(self):
 
         shutil.rmtree(root_dir)
 
+    def test_import_after_renaming_xml_data(self):
+        """
+        Test that import works fine on split mongo after renaming the blocks url.
+        """
+        content_store = contentstore()
+        split_store = modulestore()._get_modulestore_by_type(ModuleStoreEnum.Type.split)
+        course_before_rename = import_from_xml(split_store, self.user.id, TEST_DATA_DIR, ['course_before_rename'], create_course_if_not_present=True)
+        course_after_rename = import_from_xml(split_store, self.user.id, TEST_DATA_DIR, ['course_after_rename'], create_course_if_not_present=True)
+        all_items = split_store.get_items(course_after_rename[0].id, qualifiers={'category': 'chapter'})
+        for item in all_items:
+            if item.location.block_id == 'renamed_chapter':
```

Comment:
why are these in conditionals? Shouldn't we do something like 
```
renamed_chapter = [item for item in all_items if item.location.block_id == 'renamed_chapter'][0]
```
To make sure that this block is actually in `all_items`. Otherwise, we can imagine a scenario where these tests aren't actually run